metaci/plan/models.py

```diff
@@ -205,3 +206,49 @@ def get_absolute_url(self):
     @property
     def should_run(self):
         return self.active and self.plan.active
+
+
+class PlanRepositoryTriggerQuerySet(models.QuerySet):
+    def should_run(self):
+        return self.filter(active=True, target_plan_repo__active=True)
+
+
+class PlanRepositoryTrigger(models.Model):
+    source_plan_repo = models.ForeignKey(
+        PlanRepository, on_delete=models.CASCADE, related_name="Sources"
+    )
+    target_plan_repo = models.ForeignKey(
+        PlanRepository, on_delete=models.CASCADE, related_name="Triggers"
+    )
+    branch = models.CharField(max_length=255)
+    active = models.BooleanField(default=True)
+
+    objects = PlanRepositoryTriggerQuerySet.as_manager()
+
+    class Meta:
+        ordering = ["target_plan_repo", "source_plan_repo"]
+        unique_together = ("target_plan_repo", "source_plan_repo")
```

Comment:
Hmm. What if we want to trigger builds for multiple branches in the target repository?

Reply:
Good question. I hadn't considered the use case. I can add `branch` to `unique_together`.